README.md

```diff
@@ -20,12 +20,18 @@ For Rails 3, please use version 1 of Paranoia:
 gem "paranoia", "~> 1.0"
 ```
 
-For Rails 4 or 5, please use version 2 of Paranoia:
+For Rails 4, please use version 2 of Paranoia:
 
 ``` ruby
 gem "paranoia", "~> 2.0"
 ```
 
+For Rails 5, please use the "core" branch of Paranoia:
+
+``` ruby
+gem 'paranoia', github: 'rubysherpas/paranoia', :branch => 'core'
```

Comment:
Shouldn't this be `gem 'paranoia', github: 'rubysherpas/paranoia', branch: 'core'`?